### PR TITLE
jira -> issue URL

### PR DESCRIPTION
**Pull request by lwsanty**
close [[DEV-1810]]

Signed-off-by: lwsanty <lwsanty@gmail.com>

[DEV-1810]: https://athenianco.atlassian.net/browse/DEV-1810

### CODE REVIEW

File: jira_fetcher.go [hunk 1]
```diff
@@ -927,6 +928,12 @@ func convertIssue(i gojira.Issue, e *types.FetchEvent) (*types.Issue, error) {
 	issue.CommentersIDs = commentersIDs
 	issue.CommentersDisplayNames = commentersDisplayNames
 
+	selfURL, err := url.Parse(i.Self)
+	if err != nil {
+		return nil, err
+	}
+	issue.URL = fmt.Sprintf("%s://%s/browse/%s", selfURL.Scheme, selfURL.Hostname(), i.Key)
```
Comment (dennwc) [inline jira_fetcher.go:935]: Can you please add a note why we need to rebuild the URL? Is it pointing to the API endpoint or something?

Comment (lwsanty) [reply jira_fetcher.go:935]: no, jira api does not return us "browse urls", but they are required by api and web-app according to [DEV-1810](https://athenianco.atlassian.net/browse/DEV-1810), so we need to craft them

Comment (lwsanty) [reply jira_fetcher.go:936]: note added.

**Review by vmarkovtsev — APPROVED** [review]

**Review by dennwc — APPROVED** [review]
Will merge after CI.

**Pull request merged**